docker-compose-9.6-2.3.yml

```diff
@@ -0,0 +1,55 @@
+version: "2"
+
+volumes:
+  postgres:
+    external:
+      name: ecoroofs_postgres_data
```

Comment:
What do you think about using a different name here to avoid a clash? `ecoroofs_postgres96_data`?